backend/core/settings.py

```diff
@@ -23,7 +23,7 @@
 # ALLOWED_HOSTS = os.environ.get('DJANGO_ALLOWED_HOSTS').split(' ')
 
 CORS_ORIGIN_ALLOW_ALL = False
-CORS_ORIGIN_WHITELIST = ['http://localhost:3000', ]
+CORS_ORIGIN_WHITELIST = ['http://localhost:3000','https://merch-site.netlify.app' ]
```

Comment:
let's add `https://deploy-preview-*--merch-site.netlify.app` so we could test deploy previews as wll

Reply:
@ChaudharyRaman